conf/layer.conf

```diff
@@ -16,4 +16,4 @@ BBFILE_PRIORITY_webkit = "7"
 # do not error out on bbappends for missing recipes
 BB_DANGLINGAPPENDS_WARNONLY = "true"
 
-LAYERSERIES_COMPAT_webkit = "rocko sumo thud warrior zeus dunfell"
+LAYERSERIES_COMPAT_webkit = "sumo warrior zeus dunfell"
```

Comment:
Why do you delete thud but leave sumo? https://wiki.yoctoproject.org/wiki/Releases

Reply:
oops,  It's my bad. I will fix it.

Reply:
thanks for the review ... BTW